### PR TITLE
Added Greek locale files (el)

### PR DESCRIPTION
**Pull request by teoulas**
Support for Greek.

Greek has genders and some formats ("D MMMM" and "DD MMMM") should output the month in genitive.

### CODE REVIEW

**Comment by LeshaKoss** [comment]
@teoulas thanks a lot! 🌟

> "D MMMM" and "DD MMMM"

What about "Do MMMM"?

File: src/locale/el/build_format_locale/index.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  // Generate ordinal version of formatters: M -> Mo, D -> Do, etc.
+  var ordinalGenders = {
+    'M': 'ος',
+    'D': 'η',
+    'DDD': 'η',
+    'd': 'η',
+    'Q': 'ο',
+    'W': 'η'
+  }
+  Object.keys(ordinalGenders).map(function (formatterToken) {
```
Comment (LeshaKoss) [inline src/locale/el/build_format_locale/index.js]: @teoulas Please don't use `Object.keys` (it doesn't work in IE8, sadly). Separate array which is a copy of `Object.keys(ordinalGenders)` is ok

Comment (teoulas) [inline src/locale/el/build_format_locale/index.js]: IE8? Wow, I completely forgot that! I'll change it.

**Comment by teoulas** [comment]
Good point about "Do MMMM". It's not a common format, but momentjs supports it correctly. I will take a look.

**Comment by LeshaKoss** [comment]
@teoulas 

> Good point about "Do MMMM". It's not a common format, but momentjs supports it correctly. I will take a look.

In Russian locale we made it like this:

https://github.com/date-fns/date-fns/blob/master/src/locale/ru/build_format_locale/index.js#L75

**Comment by teoulas** [comment]
@LeshaKoss thanks for the tip, that's exactly how I implemented it as well!

**Comment by LeshaKoss** [comment]
@teoulas great work! Please fix the linting problem and we're good to go https://travis-ci.org/date-fns/date-fns/builds/183575458#L230

**Comment by teoulas** [comment]
Fixed linting and squashed commits. Good to go. 🚀

**Comment by LeshaKoss** [comment]
@teoulas thank you for your contribution! 🇬🇷 Greek locale was released with [1.19.0](https://github.com/date-fns/date-fns/releases/tag/v1.19.0) (diff: https://github.com/date-fns/date-fns/compare/v1.18.0...v1.19.0)

**Comment by teoulas** [comment]
👍  Thanks for the super quick response and new release!

**Pull request merged**